var/spack/repos/builtin/packages/rocm-validation-suite/package.py

```diff
@@ -38,6 +38,7 @@ class RocmValidationSuite(CMakePackage):
 
     depends_on('cmake@3.5:', type='build')
     depends_on('zlib', type='link')
+    depends_on('yaml-cpp', type='build')
```

Comment:
Why type=build? Doesn't look right to me. The cmake script seems to download/build yaml-cpp itself though? Can that be disabled so that we can use spack's yaml-cpp?